dali/test/python/test_dali_cpu_only.py

```diff
@@ -805,6 +969,38 @@ def test_stack_cpu():
     for _ in range(3):
         pipe.run()
 
+def test_batch_permute_cpu():
```

Comment:
why not using `check_single_input` here?

Reply:
Done

Reply:
... or not done
I just realized it tests `batch_permutation` as well. So I would like to keep it as it is.